### DISCUSSION TITLE
Annotation editor as side panel

### DISCUSSION THREAD

**Discussion by Tenderenden** [sumatrapdfreader/sumatrapdf#3490] [Ideas]
I believe that changing "Annotations" window from a separate window into side panel that could be opened on the right side of main viewing window (similarly to bookmarks) would be nice improvement.
I often find myself using annotations to highlight parts of pdfs to make it easier to get back to while quickly scrolling through the document. It kinda bothers me that I have to switch between 2 windows to effectively use that feature since I use Sumatra in fullscreen.
What is more since both "bookmarks" and "favourites" are opened as side panels making "annotations" side panel as well would make it more coherent designwise.

**Comment by geniusadventurer** [comment]
I really agree with this idea.

**Comment by AlexShyXie** [comment]
Yes, I agree too.

**Comment by hudcap** [comment]
Issues/comments requesting this:
#2379 
#3229 
https://github.com/sumatrapdfreader/sumatrapdf/discussions/3257#discussioncomment-5121489

**Comment by Ryukee** [comment]
Agree.

**Comment by konashevich** [comment]
Totally agree! It would be great to integrate that window into the actual application.

**Comment by beancaper** [comment]
Really need this!
